aait/backend/group-2B/Application/DTOs/PostDtos/UpdatePostDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace SocialSync.Application.DTOs.PostDtos;
+
+public class UpdatePostDto
+{
+    public int Id { get; set; }
+    public int UserId { get; set; }
+    public string? Content { get; set; }
```

Comment:
Here, the only element that might be updated is the "Content." Do you think it should be nullable?

Reply:
Fixed.